cucumber/authenticators_oidc/features/authn_oidc_okta.feature

```diff
@@ -15,21 +19,26 @@ Feature: OIDC Authenticator V2 - Users can authenticate with Okta using OIDC
           - !variable client-id
           - !variable client-secret
           - !variable claim-mapping
-          - !variable state
-          - !variable nonce
           - !variable redirect-uri
+
           - !group users
 
           - !permit
             role: !group users
             privilege: [ read, authenticate ]
             resource: !webservice
     """
-    And I am the super-user
-    And I successfully set Okta OIDC V2 variables
+    And I add an Okta user
+    And I set conjur variables
+      | variable_id                               | value                                                         | environment_variable  |
```

Comment:
Have you considered having a wide data table instead to iterate over multiple test datasets in data table or is it not needed since this is a `Background` step for subsequent `Scenarios`?

Reply:
Do you mean handling Scenario Outlines?  If so, I'd still recommend we keep data as close as possible to the scenario we're validating, thus keep it in either the scenario or background if it's relevant for all scenarios in a feature.

Reply:
Yes - that is what I was referring to with data tables.  I like your approach!!